index.js

```diff
@@ -1,11 +1,18 @@
 const stringDirection = require('string-direction');
+
 /**
  * @param {string} input
  * @return {string}
  */
-module.exports = function(input) {
+
+
+exports.add = function(input) {
   if (stringDirection.getDirection(input) === 'rtl') {
     return `covfefe ${input}`;
   }
   return `${input} covfefe`;
 };
+
+exports.translate = function(input) {
+  return input.replace('coverage', 'covfefe');
+};
```

Comment:
For backwards compatiblity you'll need to do this:

```
module.exports = add;
module.exports.translate = translate;
```

so that add is still the top level export 😝